testing/libsoundio/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: Mark Jynx <markjynx@cock.li>
+pkgname="libsoundio"
+pkgver="2.0.0"
+pkgrel=0
+pkgdesc="lightweight abstraction over various sound drivers"
+url="https://libsound.io"
+arch="all"
+options="!check"  # No test suite.
+license="MIT"
+depends="alsa-lib-dev"
```

Comment:
i doubt this should be in depends. Most likely should be in makedepends

Reply:
At compile time, libsoundio checks which backends are available (ALSA, PulseAudio, JACK...) and compiles against them. So at least one of them should be there.

Perhaps change alsa-lib-dev to alsa-lib? I don't think that the headers are needed.